README.md

```diff
@@ -7,5 +7,226 @@
 [![PyPI - Python Version](https://img.shields.io/pypi/pyversions/graviti)](https://pypi.org/project/graviti/)
 [![Downloads](https://pepy.tech/badge/graviti/month)](https://pepy.tech/project/graviti)
 
-Graviti Python SDK is a python library to access Graviti Data Platform and manage your datasets.
-It provides a pythonic way to access your datasets by Graviti OpenAPI.
+Graviti Python SDK is a python library to access [Graviti](https://www.graviti.com) workspace and
```

Comment:
The content `README.md` is similar to quick start in sphinx doc, is it possible to reuse the rst or md file?

Reply:
The two files are not exactly the same, `README.md` has something more. Secondly, I haven't found a good way to use `README.md` in `docs/source/index`. Maybe we can optimize it later.